Source/WebGPU/WebGPU/Texture.h

```diff
@@ -162,6 +163,16 @@ class Texture : public WGPUTextureImpl, public RefCounted<Texture>, public CanMa
     mutable WeakHashSet<CommandEncoder> m_commandEncoders;
     id<MTLSharedEvent> m_sharedEvent { nil };
     uint64_t m_sharedEventSignalValue { 0 };
-};
+} SWIFT_SHARED_REFERENCE(retainTexture, releaseTexture);
 
 } // namespace WebGPU
+
+inline void retainTexture(WebGPU::Texture* obj)
+{
+    WTF::retainRefCounted(obj);
+}
+
+inline void releaseTexture(WebGPU::Texture* obj)
+{
+    WTF::releaseRefCounted(obj);
```

Comment:
deref(obj)

Reply:
Have implemented all your suggestions in this area.